utils/logger.js

```diff
@@ -0,0 +1,109 @@
+const createLogger = () => {
+  const isProdMode =
+    process.env.NODE_ENV === "prod" || process.env.NODE_ENV !== "development"
+      ? true
+      : false;
+
+  //TODO: move this var to env file.
+  // level 1 = debug
```

Comment:
Can we use explicit string log levels instead of numerical ones?
Ideally we'd have some kind of enumeration but that's not something we can easily set in an env; seems like in absence of that explicit string levels are a less mysterious choice.

Reply:
True. That more clear. Will update. 

I actually converted this from my typescript version. 
It was an enum.